### PR TITLE
Feature/switch login functionality associations

### PR DESCRIPTION
**Pull request by Andresdev02**
CLBV-285

### CODE REVIEW

**Review by lagartoverde — CHANGES_REQUESTED** [review]
If you want to build on top of another PR please select the target branch correctly so it only shows the changes of this PR

**Comment by Andresdev02** [comment]
@lagartoverde I changed the base branch.

File: app/routes/auth/logout.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ export default class AuthLogoutRoute extends Route {
         let wasMockLoginSession = this.session.isMockLoginSession;
         await this.session.invalidate();
         let logoutUrl = wasMockLoginSession
-          ? this.router.urlFor('mock-login')
+          ? this.router.urlFor('login')
```
Comment (lagartoverde) [inline app/routes/auth/logout.js:15]: If the session was a mock session wouldn't it need to redirect to mock-login

**Review by lagartoverde — APPROVED** [review]
Overall looks good, just a small remark

File: app/routes/auth/logout.js (hunk 1 above)
Comment (Andresdev02) [reply app/routes/auth/logout.js]: @lagartoverde I changed it back to mock-login.

**Review by lagartoverde — APPROVED** [review]

**Pull request merged**